stm/stm.ml

```diff
@@ -1339,7 +1307,6 @@ end = struct (* {{{ *)
   type response =
     | RespBuiltProof of Proof_global.closed_proof_output * float
     | RespError of error
-    | RespStates of (Stateid.t * State.partial_state) list
```

Comment:
The entire logic for sending back a state is gone. Are you saying that with this patch, if one edits a broken proof, all steps frmo the beginning of the proof to the error are recomputed?

Reply:
I don't know @gares , I did that as I understood from our conversation that this was a non-very important optimization. I can resurrect that part without a problem, but will have to tweak the interface a bit so workers declare local persistent state.

Reply:
What I mean was "always send back state(s) as part of the response to the other message"

Reply:
Ah! Understood, thanks.